scheme-service/src/main/java/uk/gov/justice/laa/fee/scheme/exceptions/GlobalExceptionHandler.java

```diff
@@ -26,4 +26,32 @@ public ResponseEntity<ErrorResponse> handleCategoryOfLawNotFound(CategoryCodeNot
 
     return new ResponseEntity<>(errorResponse, HttpStatus.NOT_FOUND);
   }
+
+  /**
+   * Global exception handler for FeeSchemeNotFoundForDateException exception.
+   */
+  @ExceptionHandler(FeeSchemeNotFoundForDateException.class)
+  public ResponseEntity<ErrorResponse> handleFeeSchemeNotFoundForDate(FeeSchemeNotFoundForDateException ex) {
```

Comment:
Can we combine these due to repeated code, something like:
```suggestion
  @ExceptionHandler({FeeSchemeNotFoundForDateException.class, FeeEntityNotFoundException.class})
  public ResponseEntity<ErrorResponse> handleNotFound(Exception ex) {
```